tests/contract_testcases/substrate/functions/virtual_function_call_02.sol

```diff
@@ -0,0 +1,19 @@
+interface I {
+    function total() external returns (int256);
+}
+
+contract A {
+    function total() external returns (int256) {
+        return 4;
+    }
+}
+
+contract B is I, A {
+    function x() public {
+        int256 total = total();
+    }
+}
+
+// ---- Expect: diagnostics ----
+// error: 13:24-31: functions declared external cannot be called via an internal function call
+// 	note 6:5-47: declaration of function 'total'
```

Comment:
What were you trying to test in this contract?
These errors do not look like they are related to your changes.

Reply:
It ensures functions without a body are[ only allowed on abstract contracts](https://github.com/hyperledger/solang/pull/1315#pullrequestreview-1433941958)

Reply:
That makes sense.
Do you need the `external` keyword in those functions? It might be masking other errors in this file, but your goal is for it to have no errors.

Reply:
Interface functions must be declared `external`